content/en/docs/collector/internal-telemetry.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Internal Telemetry
+weight: 25
+cSpell:ignore: journalctl kube otecol pprof tracez zpages
+---
+
+The Collector offers multiple ways to measure and monitor its own health. In
+this section, you'll learn how to enable internal observability.
+
+## Enabling observability internal to the Collector
+
+By default, the Collector exposes service telemetry in two ways:
+
+- Internal [metrics](#configuring-metrics) are exposed using a Prometheus
+  interface which defaults to port `8888`.
+- [Logs](#configuring-logs) are emitted to `stdout`.
+
+[Traces](#configuring-traces) are not exposed by default but two feature gates
+offer experimental support for a configuration based on the OpenTelemetry
+Configuration schema.
+
+### Configuring metrics
```

Comment:
```suggestion
### Configure internal metrics
```

Following Google's style guide, better go the plain or base form: https://developers.google.com/style/headings